dev-util/maturin/maturin-1.5.0.ebuild

```diff
@@ -493,11 +495,12 @@ src_configure() {
 		--features full,password-storage$(usev ssl ,native-tls)
 	)
 
-	export MATURIN_SETUP_ARGS=${cargoargs[*]}
+	cargo_src_configure "${cargoargs[@]}"
```

Comment:
One last thing, drop the `$(usev debug '--profile dev')` from cargoargs, cargo_src_configures handles that for us.

After that, feel free to merge, lgtm.

Reply:
Actually wait, there's one more thing.

Should replace the whole cargoargs array:
```
local myfeatures=(
	# like release.yml + native-tls for better platform support than rustls
	full
	password-storage
	$(usev ssl native-tls)
)
```
And drop passing the array entirely, I'd move OPENSSL_NO_VENDOR=1 at the start of src_configure too given it feels odd to run it after cargo_src_configure (not that it matters).

I could fix nitpicks after myself if need be though.